pandana/utils.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+
+def reindex(series1, series2):
+    """
+    Reindex the first series by the second series.
+
+    Parameters
+    ----------
+    series1 : pandas.Series
+        Pandas series to reindex
+    series2 : pandas.Series
+        Pandas series to set the index of series1 by
+
```

Comment:
Agreed, made the change with commit https://github.com/UDST/pandana/pull/65/commits/b3c9d1cfc76025114b8a7214c29aa4d53d7ef7f4